tests/components/habitica/test_todo.py

```diff
@@ -627,9 +635,12 @@ async def test_move_todo_item(
     uid: str,
     second_pos: str,
     third_pos: str,
+    fixture: str,
 ) -> None:
     """Test move todo items."""
-
+    habitica.reorder_task.return_value = HabiticaTaskOrderResponse.from_json(
```

Comment:
we should also check that `self.coordinator.data.user.tasksOrder.todos/dailies` was updated, if we want to introduce that change